kep/61-real-load-aware-scheduling/61-real-load-aware-scheduling.md

```diff
@@ -216,7 +220,7 @@ Following is the algorithm:
 3. Calculate the score of the current node for each type of resource: <img src="https://render.githubusercontent.com/render/math?math=S_i = M %2B r %2B V">
 4. Get a score for each type of resource and bound it to [0,1]: <img src="https://render.githubusercontent.com/render/math?math=S_i = \min(S_i, 1.0)">
 5. Calculate the node priority score per resource as: <img src="https://render.githubusercontent.com/render/math?math=U_i = (1 - S_i) \times MaxPriority ">
-6. Get the final node score as: <img src="https://render.githubusercontent.com/render/math?math=U = \min(U_i)">
+6. Get the final node score as: <img src="https://render.githubusercontent.com/render/math?math=U =\min(U_i)">
```

Comment:
This is for discussion: https://github.com/kubernetes-sigs/scheduler-plugins/pull/61#discussion_r502648458